atom/browser/native_window_views.cc

```diff
@@ -1264,11 +1276,37 @@ void NativeWindowViews::OnWidgetMove() {
 void NativeWindowViews::HandleKeyboardEvent(
     content::WebContents*,
     const content::NativeWebKeyboardEvent& event) {
+#if defined(OS_MACOSX)
+  if (event.GetModifiers() == blink::WebInputEvent::kMetaKey) {
+    if (event.windows_key_code == ui::VKEY_OEM_4)
```

Comment:
Can you add comments explaining what `VKEY_OEM_4` and `VKEY_OEM_6` represent?

Reply:
I finally got a chance to test this on macOS and it doesn't work. Turns out they don't support the extra mouse buttons by default, not sure what this is then as I took it from chrome under the assumption that it was the way to refer to the mouse forward and back keys on macOS. I'll remove this.